### PR TITLE
Db update to reflect 2.7.0

### PR DESCRIPTION
**Pull request by srugano**

### CODE REVIEW

File: src/hope_country_report/api/serializers.py [hunk 1]
```diff
@@ -25,9 +25,8 @@ class Meta:
 
     @cached_property
     def selected_office(self) -> CountryOffice:
-        co_slug: str = self.context["view"].kwargs.get(self.co_key)
+        co_slug: str = self.context["view"].kwargs[self.co_key]
```
Comment (domdinicola) [inline src/hope_country_report/api/serializers.py:28]: why?

File: src/hope_country_report/apps/power_query/processors.py [hunk 2]
```diff
@@ -277,7 +276,6 @@ def insert_external_image(self, document: fitz.Document, field_name: str, image_
             logger.error(f"No valid rectangle or page index found for field {field_name}. Cannot insert image.")
             return
         page = document[page_index]
-        logger.info(f"Inserting image {image_path} into field {field_name} on page {page_index}")
```
Comment (domdinicola) [inline src/hope_country_report/apps/power_query/processors.py:280]: why?

Comment (srugano) [reply src/hope_country_report/apps/power_query/processors.py:280]: Not needed.

File: src/hope_country_report/api/serializers.py (hunk 1 above)
Comment (srugano) [reply src/hope_country_report/api/serializers.py]: addressed

**Pull request merged**